### PR TITLE
Dom task

### PR DESCRIPTION
**Pull request by AriadnaKravchuk**
### DOM

[Demo](https://ariadnakravchuk.github.io/DOM) |
[Code base](https://github.com/AriadnaKravchuk/DOM)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by AriadnaKravchuk** [comment]
I completed self-checks

File: submissions/AriadnaKravchuk/dom/script.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    );
+};
+
+const changeContent = function(event) {
+    const character = characters.find((character) => character.id === event.target.id);
+    mainInner.innerHTML = createContent(character);
+};
+
+navList.addEventListener("click", changeContent);
+mainInner.innerHTML = createContent(characters[0]);
```
Comment (madmaxWMFU) [inline submissions/AriadnaKravchuk/dom/script.js:58]: Set a variable with an initial value.

File: submissions/AriadnaKravchuk/dom/script.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+]
+
+const navList = document.querySelector(".nav__list");
+const mainInner = document.querySelector(".main__inner");
+
+function createContent(character) {
```
Comment (madmaxWMFU) [inline submissions/AriadnaKravchuk/dom/script.js:39]: Read about [destruction](https://javascript.info/destructuring-assignment). Also, use that you can select the first element.

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Excellent! See some comments below.

File: submissions/AriadnaKravchuk/dom/script.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    }
+]
+
+const navList = document.querySelector(".nav__list");
+const mainInner = document.querySelector(".main__inner");
+const [firstPage] = characters
```
Comment (madmaxWMFU) [inline submissions/AriadnaKravchuk/dom/script.js:38]:
```suggestion
const [firstPage] = characters;
```

Use [prettier](https://prettier.io/)

**Review by madmaxWMFU — APPROVED** [review]
Great, I approve of your work. Good luck with future tasks!

**Comment by AriadnaKravchuk** [comment]
> Great, I approve of your work. Good luck with future tasks!

Thanks!

**Pull request merged**